docs/src/proposals/multiple-gossip-identities.md

```diff
@@ -0,0 +1,61 @@
+# Multiple Gossip Identities
```

Comment:
I think this proposal is more about "Validator Groups" -- multiple validator (ie, `solana-validator`) instances  are independent from a gossip perspective, but are all tied to the same vote account.  Only one in each validator group is the block producer when it's _their_ turn in the leader schedule, and only one validator in the group will ever vote on any given slot.